decidim-conferences/app/forms/decidim/conferences/admin/conference_form.rb

```diff
@@ -57,10 +60,41 @@ def scope
           @scope ||= current_organization.scopes.find_by(id: scope_id)
         end
 
+        def processes_for_select
+          @processes_for_select ||= Decidim.find_participatory_space_manifest(:participatory_processes)
+                                           .participatory_spaces.call(current_organization)&.order(title: :asc)&.map do |process|
+            [
+              translated_attribute(process.title),
+              process.id
+            ]
+          end
+        end
+
+        def assemblies_for_select
+          @assemblies_for_select ||= Decidim.find_participatory_space_manifest(:assemblies)
+                                            .participatory_spaces.call(current_organization)&.order(title: :asc)&.map do |assembly|
+            [
+              translated_attribute(assembly.title),
+              assembly.id
+            ]
+          end
+        end
+
+        def consultations_for_select
+          @consultations_for_select ||= Decidim.find_participatory_space_manifest(:consultations)
```

Comment:
Same, what happens with this `collect` if `decidim-consultations` is not installed?

Reply:
The same as I said before